backend/src/test-integration/java/com/crm/api/v1/AuthenticationControllerIT.java

```diff
@@ -0,0 +1,114 @@
+package com.crm.api.v1;
+
+import com.crm.App;
+import com.crm.BaseIntegrationTest;
+import com.crm.exception.ElementAlreadyExistException;
+import com.crm.exception.InvalidCredentialsException;
+import com.crm.exception.UserDisabledException;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static com.crm.exception.ErrorDict.ACCOUNT_ALREADY_EXIST_BY_EMAIL;
+import static com.crm.exception.ErrorDict.ACCOUNT_ALREADY_EXIST_BY_LOGIN;
+import static com.crm.exception.ErrorDict.INVALID_CREDENTIALS;
+import static com.crm.exception.ErrorDict.USER_DISABLED;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertTrue;
+import static org.springframework.http.HttpStatus.CONFLICT;
+import static org.springframework.http.HttpStatus.FORBIDDEN;
+import static org.springframework.http.HttpStatus.OK;
+import static org.springframework.http.HttpStatus.UNAUTHORIZED;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest(classes = App.class)
+@AutoConfigureMockMvc
+public class AuthenticationControllerIT extends BaseIntegrationTest {
+
+    @Autowired
+    private MockMvc mvc;
+
+    @Test
+    void shouldReturn_Ok_StatusWhenCallingAuthEndpoint() throws Exception {
+        mvc.perform(post("/auth")
+                .content("{\n" +
```

Comment:
Do not hardcode body. Either create a separate json file with the body and parse it using `objectMapper.readValue()` or crate an object AccountRequest and use smth like: `final String body = objectMapper.writeValueAsString(accountRequest);`